parser.py

```diff
@@ -479,14 +217,16 @@ def uop(self, *args):
 
         if op_name == "CX":
             params = [None]
-            wires = [x.list for x in extra_args]
+            wires = flatten(extra_args)
```

Comment:
I wouldn't mind if we could also remove the recursive flattening. It seems uneeded, I'm sure we could find a workaround